internal/output/output.go

```diff
@@ -50,7 +50,15 @@ type Output struct {
 type OutputOpts struct {
 	// ForceColor ignores all terminal detection and enabled coloured output.
 	ForceColor bool
-	Verbose    bool
+	// ForceTTY ignores all terminal detection and enables TTY output.
+	ForceTTY bool
+
+	// ForceHeight ignores all terminal detection and sets the height to this value.
+	ForceHeight int
+	// ForceWidth ignores all terminal detection and sets the width to this value.
+	ForceWidth int
```

Comment:
Obviously don't do it now, but these _might_ actually be something we'd want to expose at some point for more than just testing.

Reply:
Yeah, had the same thought!